src/src/Commands/ListCommand.php

```diff
@@ -36,32 +36,43 @@ protected function configure() {
 		$test_types_list = implode( ', ', $this->environment->get_cache()->get_manager_sync_data( 'test_types' ) );
 		$this
 			->setDescription( 'List test runs.' )
+			->addOption( 'extensions', 'e', InputOption::VALUE_OPTIONAL, '(Optional) Retrieve results for these extensions (Accepts slugs or IDs, comma-separated).' )
```

Comment:
Minor suggested tweak:

```suggestion
			->addOption( 'extensions', 'e', InputOption::VALUE_OPTIONAL, '(Optional) Retrieve results for these extensions (Accepts either a comma separated list of slugs or IDs).' )
```

Only because as it's currently worded someone might think they could do something like this:

```
php qit-cli.php list-tests --extensions=woocommerce-product-bundles,woocommerce,4652610 --test_types=api
```

Which will only run the list against the ID.

Other than that, this tested great and I think it's good to go!

<img width="1046" alt="Screenshot 2023-12-15 at 8 48 55 AM" src="https://github.com/woocommerce/qit-cli/assets/71906536/8c7d619d-d1c3-4cc6-a563-d96bda9ba310">

Reply:
It actually accepts both IDs and slugs
![image](https://github.com/woocommerce/qit-cli/assets/9341686/8ffc809e-cae7-43f0-a9da-b434813840e7)

Reply:
Ah, thanks, I switched to prod and it worked, so much be something odd in staging that was causing that behavior.